test/zzz-rsyslog.sh

```diff
@@ -30,7 +30,7 @@ cleanup() {
     sudo journalctl -u $rsyslog_service --since="-1hour" > $ARTIFACT_DIR/rsyslog-rsyslog.log 2>&1
     if [ -n "${rsyslog_save}" -a -d "${rsyslog_save}" ] ; then
         sudo rm -rf /etc/rsyslog.d/*
```

Comment:
Should this and the raw bellow use the path from the params as well?

Reply:
Ok.  I'm updating the test script zzz-rsyslog.sh to generate vars file to sync the parameter.